cpp/benchmarks/string/factory.cu

```diff
@@ -63,7 +63,7 @@ static void BM_factory(benchmark::State& state)
 
   for (auto _ : state) {
     cuda_event_timer raii(state, true, cudf::get_default_stream());
-    cudf::make_strings_column(pairs);
+    cudf::make_strings_column(pairs, cudf::get_default_stream());
```

Comment:
will previous thrust call on `thrust::device` use the same stream as `cudf::get_default_stream()` ?

Reply:
The default value on the `stream` argument does not seem to be removed in this PR. should it be included too?

Reply:
No, removing default values only applies to detail functions. We'll be addressing the stream parameter in public APIs like this one later.